packages/docusaurus-mdx-loader/src/remark/utils/index.ts

```diff
@@ -154,3 +155,32 @@ export function assetRequireAttributeValue(
     },
   };
 }
+
+/**
+ * Similar to stringify-object, but keeps spread operators,
+ * instead of turning them into strings.
+ * @param objects
+ */
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export function constructArrayString(objects: any[]): string {
+  let result = '[';
+
+  for (const obj of objects) {
+    if (typeof obj === 'string') {
+      result = `${result}\n\t${obj},`;
+    } else {
+      result = `${result}\n\t${stringifyObject(obj).replace(/\n/g, '\n\t')},`;
+    }
+  }
+
+  // Remove trailing coma
+  result = result.replace(/,$/, '');
+  // Close array on same line if it is empty
+  if (result === '[') {
+    result += ']';
+  } else {
+    result += '\n]';
+  }
+
+  return result;
+}
```

Comment:
wonder if we shouldn't use something cleaner for that instead of string concat.

For example: https://github.com/syntax-tree/estree-util-to-js

Reply:
I removed this whole function as per @wooorm's suggestion. But in case we need `value` to be set, this can be done with [astring](https://github.com/davidbonnet/astring), which seems to do the job well.